Packs/CommonScripts/Scripts/StixCreator/StixCreator.py

```diff
@@ -95,75 +176,88 @@ def main():
         else:
             value = all_args[indicator_fields].get('value', '')
 
-        demisto_score = all_args[indicator_fields].get('score', '').lower()
+        if demisto_indicator_type in XSOAR_TYPES_TO_STIX_SCO and is_sco:
+            stix_type = XSOAR_TYPES_TO_STIX_SCO.get(demisto_indicator_type)
+            stix_id = create_sco_stix_uuid(all_args[indicator_fields], stix_type, value)
```

Comment:
Since we use the indicator_fields often, let's extract this variable. before the first usage on line 172
like that:
```python
indicator_fields = all_args[indicator_fields]
```